Uno.Gallery/Uno.Gallery.Shared/Entities/SampleCategory.cs

```diff
@@ -15,6 +15,6 @@ public enum SampleCategory
 		Theme,
 		Components,
 		Features,
-
+		Toolkit,
```

Comment:
Can we have two sections under Components ? 
> Components
    > WinUI controls
    > Toolkit controls

Reply:
this would requires some refactoring, 
since we will be moving from 2-tiers navigation to 3 tiers
which deserves to be in its own pr&/issue.